.travis/bootstrap_catdog.sh

```diff
@@ -20,4 +22,4 @@ cp ../plugin_template/.travis/post_before_script.sh .travis/
 # generate a git commit
 git init
 git add .
-git commit -m $'Initial commit\n\n[noissue]'
+git commit -m "${COMMIT_MSG}"
```

Comment:
It feels like a hack having the plugin bootstrap test check plugin_template's commit messages. For one thing, I am guessing it doesn't handle multiple commits?

What if we convert templates/travis/.travis/validate_commit_message.py.j2 to templates/travis/.travis/validate_commit_message.py (the only thing we use jinja for in this file is to add a header which I don't think we need) and then write a small github workflow or travis job to run it?

Reply:
it is for doing the `Required PR` thing, and it only checks the last commit

Reply:
Don't we want to validate all commits?

Reply:
It also still feels weird to me that we're using the bootstrap test code to validate commit messages.